greedy_algorithms/DJnew.cpp

```diff
@@ -0,0 +1,149 @@
+/**
+ * @file
+ * @brief Implementation of Djikstra's Algorithm
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂